discovery-provider/src/queries/query_helpers.py

```diff
@@ -1038,27 +949,6 @@ def get_save_counts(
         )
     return save_counts_query.all()
 
-
-def get_followee_count_dict(session, user_ids):
-    # build dict of user id --> followee count
-    followee_counts = (
-        session.query(
-            Follow.follower_user_id,
-            func.count(Follow.follower_user_id)
-        )
-        .filter(
-            Follow.is_current == True,
-            Follow.is_delete == False,
-            Follow.follower_user_id.in_(user_ids)
-        )
-        .group_by(Follow.follower_user_id)
-        .all()
-    )
-    followee_count_dict = {user_id: followee_count for (
-        user_id, followee_count) in followee_counts}
-    return followee_count_dict
-
-
 def get_follower_count_dict(session, user_ids, max_block_number=None):
```

Comment:
Would this one benefit?

Reply:
The only place it's called is in notifications.py where max_block_number is always set and passed, making it not useful here.